.github/workflows/ci.yml

```diff
@@ -23,67 +23,20 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 16.x
-          cache: yarn
-      - name: Install Dependencies
-        run: yarn install --frozen-lockfile
-      - name: Lint
-        run: yarn lint
-      - name: Run Tests
-        run: yarn test:ember
-
-  floating:
-    name: "Floating Dependencies"
-    runs-on: ubuntu-latest
-    timeout-minutes: 10
-
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: pnpm/action-setup@v2
         with:
-          node-version: 16.x
-          cache: yarn
+          version: 8
       - name: Install Dependencies
-        run: yarn install --no-lockfile
+        run: pnpm install --frozen-lockfile
       - name: Run Tests
-        run: yarn test:ember
-
-  try-scenarios:
-    name: ${{ matrix.try-scenario }}
-    runs-on: ubuntu-latest
-    needs: "test"
-    timeout-minutes: 10
-
-    strategy:
-      fail-fast: false
-      matrix:
-        try-scenario:
-          - ember-lts-4.4
-          - ember-lts-4.8
-          - ember-release
-          - ember-beta
-          - ember-canary
-          - embroider-safe
-          - embroider-optimized
-
-    steps:
-      - uses: actions/checkout@v3
-      - name: Install Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16.x
-          cache: yarn
-      - name: Install Dependencies
-        run: yarn install --frozen-lockfile
-      - name: Run Tests
-        run: ./node_modules/.bin/ember try:one ${{ matrix.try-scenario }}
-
+        run: pnpm test
   automerge:
-    needs: [test, floating, try-scenarios]
```

Comment:
@AmauryD can we add back the ember-try scenarios please?

Reply:
is it necessary ? It's already tested by the test command. https://github.com/AmauryD/ember-flatpickr/blob/main/test-app/config/ember-try.js

https://github.com/AmauryD/ember-flatpickr/blob/6afd587d5806042b123cfe44531b398cede8485c/test-app/package.json#L35

Reply:
The ember-try setup should remain as close to how it was before as possible for ease of upgrading.

Reply:
Also, I don't know if it runs in CI at all now. We want each try scenario to be a separate run so we can see which ones pass and fail.

Reply:
I understand. I'll fix this after work 😄

Reply:
Thank you for all your hard work on this!

Reply:
The scenarios are back !